sites/public/pages/listings/page/[page].tsx

```diff
@@ -38,15 +38,28 @@ export default function ListingsPage(props: ListingsProps) {
 
 export async function getStaticPaths(context: { locales: Array<string> }) {
   try {
-    const response: PaginatedListings = await axios.get(process.env.listingServiceUrl, {
-      params: { page: "1", limit: "10" },
-    })
+    const response: AxiosResponse<PaginatedListings> = await axios.get(
+      process.env.listingServiceUrl,
+      {
+        // TODO: Think about making this limit configurable.
+        params: { page: "1", limit: "10" },
+      }
+    )
 
+    /**
+     * We want to return an array inside paths that contains an entry for each combination
+     * of locale and page number. So we create an array that goes from 1 .. totalPages and
+     * map that with locales.
+     */
     return {
-      paths: context.locales.flatMap((locale: string) => ({
-        params: { page: response.meta.totalPages },
-        locale: locale,
-      })),
+      paths: context.locales.flatMap((locale: string) =>
+        Array.from({ length: response.data.meta.totalPages }, (_, index) => index + 1).map(
```

Comment:
I don't know if there's a polyfill elsewhere that fixes this, but IE doesn't support `Array.from()`:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#browser_compatibility

Reply:
I don't think we're going to actually use these pages, so I'll just merge it so that it will at least be correct and render. If we do want to use these, we can just create an array with numbers manually instead of using `Array.from()`